src/main.ts

```diff
@@ -25,7 +25,7 @@ export default async function(api: (token: string) => GitHub): Promise<void> {
   const externalToken = process.env.COMMITTER_TOKEN || ''
 
   const [owner, repo] = getInput('homebrew-tap', { required: true }).split('/')
-  const formulaName = getInput('formula-name', { required: true })
+  const formulaName = getInput('formula-name', { required: true }).replace(/^.*\//, '')
```

Comment:
If it's desirable to allow a user to submit a formula-name that contains a slash (I can't imagine when this would be desirable, but 🤷‍♂ ) we could do the fallback here in code rather than setting the default within the action.yml.

Something like `const formulaName =  getInput('formula-name') || context.repository.replace(/^.*\//, '')`